sktime/forecasting/base/adapters/_pytorch.py

```diff
@@ -56,8 +63,6 @@ def _fit(self, y, fh, X=None):
         X : iterable-style or map-style dataset
             see (https://pytorch.org/docs/stable/data.html) for more information
         """
-        from sktime.forecasting.base import ForecastingHorizon
```

Comment:
it might be safer to have it in here to avoid cirular imports?

Reply:
I would be very surprised if something from base.py is importing something directly or indirectly from _pytorch.py

If this would be the case, I think this would indicate some design issues since something from a base module is importing something that is specific to a certain class. Thus, if this would be the case, we should factor the imported component from the specific module out.

Reply:
makes sense.